package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-commercetools",
-  "version": "3.2.2",
+  "version": "3.3.0",
```

Comment:
Really? How is the code of users changing? They just update the version, install the dependencies and are good to go right?